### DISCUSSION TITLE
Pyramid Framework

### DISCUSSION THREAD

**Discussion by TrailBlazor** [zauberzeug/nicegui#3119] [Q&A]
### Question

Hi,

Is it possible to use NiceGui within the Pyramid Framework?  
Thanks

**Comment by rodja** [comment]
Not at the moment. We use FastAPI as web framework. It may be possible to refactor the code to make it replaceable as I wrote in the discussion about switching to Litestar (#1351).

**Comment by hasansezertasan** [comment]
Pyramid seems to be a WSGI framework. So you could try [a2wsgi](https://github.com/abersheeran/a2wsgi) to make NiceGUI work with Pyramid.
